src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
```

Comment:
The href path './styles/style.css' should be checked to ensure it matches the correct directory structure. Previously, there was a typo in the path ('./bloks/' instead of './blocks/'). Make sure this path is correct and matches your project's folder structure.